Payments/UserInterface/01a_ListAuthorizations.md

```diff
@@ -0,0 +1,299 @@
+
+- *Connection*   
+   Name of the Connection. Note that this is the customer-specific name that has been given the connection. For detailed information, see [Establish connections to payment service providers (PSP)](../Integration/01_EstablishConnection.md).
+
+- *Created by*   
+   User name of the person or API who has created the transaction. Can be a real user or an API user.
+
+- *Modified by*   
+   User name of the person or API who has changed the transaction. Can be a real user or an API user.
+
+- *ID*   
+   Actino unique identifier of the transaction.   
+     
+- *Type*   
+   Kind of the displayed transaction, in this case an authorization transaction.  
+
+- *Connection ID*   
+   Connection identification number. The ID number is automatically assigned by the system.    
+
+- *Created At*   
+   Date and time, on which the payment service provider has created the transaction. 
+
+- *Modified At*   
+   Date and time, on which the payment service provider has modified the transaction.  
+
+- *Currency*   
+   Currency in which the payment will be posted.  
+
+- *Amount*   
+   Invoice amount of the payment in the currency in which the payment will be posted.   
+
+- *Reference currency*   
+  Currency of the original payment. 
+
+- *Reference amount*   
+   Currency of the original payment. Makes it possible that the accounting can post the payment to the correct currency account so that a refund later do not lead to currency differences.
+
+- *Name*   
+   First name of the person who has made the payment.   
+
+- *Surname*   
+   Last name of the person who has made the payment.   
+
+- *Company*   
+   Name of the company on whose behalf the payment was made.   
+
+- *Address*   
+   Street and house number of the address.<!---Straße und Hausnummer? oder konkatenierte Adresse? -->     
+
+-  *Postal code*   
+   Zip code of the address.    
+
+- *City*   
+   City of the address.    
+
+- *State*   
+   State of the address.    
+
+- *Country*   
+   Country of the address.   
+
+- *Charges amount*   
+   Amount of the fee.    
+
+- *Charges currency*   
+   Currency of the fee.      
+
+- *Charges other*   
+   Other fees.     
+
+- *Reference*   
+   Mostly, depending on the payment service provider, the order number.
+
+- *Intented use*   
+   In general, depending on the payment service provider, a further description of the transaction.
+
+- *Charges intended use*   
+   Description of the fee, for example transaction fee.
+
+- *Booking date*   
+   Date on which the payment was posted.   
+
+- *Due date*  
+   If applicable, date on which the payment is to be collected.
+
+- *Payment method*   
+   The way of payment that the customer has chosen, for example, credit card, PayPal&trade;, direct debit.   
+
+- *Status information*   
+   If applicable, specific information on the status of the transaction.  
+
+- *Internal reference*   
+   Actindo specific reference back to the document in the billing document.
```

Comment:
"back to the document in the billing document" --> back to the billing document? Oder ist es Doppel aus einem Grund?

Reply:
Done